src/dialogs/locales/en.json

```diff
@@ -314,7 +314,7 @@
 	],
 	"UpdateMyEmailRetryPromptMessage": "Hmm, I'm not sure what you meant. Do you want to update this email address?",
 	"ConfirmEmailRetryPromptMessage": "Oops, looks like the email address you entered isn’t formatted correctly (here’s an example: name@mail.com). Let’s try again.",
-	"ConfirmEmailRetryExceededMessage": "I'm sorry, but it looks like I can't help you today. If you'd like, I can have a Service Canada represtentative call you on the phone to help you with your email update.\n\n Would you like me to set up a call?",
```

Comment:
this message does not have a France one, also the representative is not right

Reply:
looks ConfirmEmailRetryExceededMessage this message never used so never mind my previous comments